### PR TITLE
feat: support itemIds field in updateOrderedSetMutation

### PR DESCRIPTION
**Pull request by mc-jones**
This PR supports passing a `item_ids` field through to `gravity` when updating a set. See the sibling PR below for more details on the motivation for this change. 

### Related PR
- https://github.com/artsy/gravity/pull/16087

### CODE REVIEW

File: src/schema/v2/OrderedSet/updateOrderedSetMutation.ts [hunk 1]
```diff
@@ -43,7 +45,7 @@ interface Input {
 }
 
 const SuccessType = new GraphQLObjectType<any, ResolverContext>({
-  name: "updateOrderedSetSuccess",
+  name: "UpdateOrderedSetSuccess",
```
Comment (mc-jones) [inline src/schema/v2/OrderedSet/updateOrderedSetMutation.ts:48]: We don't [use](https://github.com/search?q=org%3Aartsy+updateOrderedSetMutation&type=code) this type yet, so not breaking.

Comment (mc-jones) [reply src/schema/v2/OrderedSet/updateOrderedSetMutation.ts:48]: Same for other similar typing.

Comment (mzikherman) [reply src/schema/v2/OrderedSet/updateOrderedSetMutation.ts:48]: Actually we do (as of..this week?!). Just one mutation in Forque - https://github.com/artsy/forque/blob/53bf9acda74840e4d8441dd892025a62d599d018/src/pages/features/mutations/addOwnerToOrderedSet.tsx#L13

It's for 'adding sets to a feature' functionality. The way that's implemented is it's actually an update on the set, setting the `owner_id` to the feature id, and `owner_type: 'Feature'`.

**Review by mzikherman — APPROVED** [review]
Once this is merged, the 'schema update' PR in Forque will likely fail due to the type name change, but that's trivially corrected then.

File: src/schema/v2/OrderedSet/updateOrderedSetMutation.ts (hunk 1 above)
Comment (mc-jones) [reply src/schema/v2/OrderedSet/updateOrderedSetMutation.ts:48]: Ah, bad searching on my part. I've updated the typing in the related `forque` pr with [this commit](https://github.com/artsy/forque/pull/692/commits/18c7f339004f22a80826b78b4835ed7aca93a6e9).

**Pull request merged**